### DISCUSSION TITLE
Rustdesk for iOS - screen sharing

### DISCUSSION THREAD

**Discussion by LarsK1** [rustdesk/rustdesk#5134] [Q&A]
Hi everyone,
I'm quite new to Rustdesk, but wanted to migrate all Anydesk-customers to it. But how can I at least see the screen of iOS devices? (In Anydesk seeing is possible, controlling unfortunately not)

**Comment by rustdesk** [comment]
close as duplicate https://github.com/rustdesk/rustdesk/discussions/4839

**Reply by meqativ** [reply]
i dont think this is duplicate of that, since this is also asking for just the Screen __Sharing__ feature, which apple does provide API for

**Reply by PC-Support-Services** [reply]
Nearly three years later and it's still not possible to view iOS devices screens. 

Both my anydesk and RustDesk licences are expiring in the next six months and I'll need to decide which one gets my money based purely on functionality...
